src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -155,7 +155,7 @@ public boolean isTemplateNull()
      */
     public boolean doesExist()
     {
-        return template == null ? false : true;
+        return !isTemplateNull();
```

Comment:
I'd say just inline that method and remove it. Then flip all ! == and be done :D